modules/input.nix

```diff
@@ -222,6 +231,7 @@ let
         TapAndDrag = touchpad.tapAndDrag;
         TapDragLock = touchpad.tapDragLock;
         ScrollMethod = touchpad.scrollMethod;
+        ScrollFactor = touchpad.scrollSpeed;
```

Comment:
Why not call it `scrollFactor` like the actual config option?

Reply:
because it's called scrollSpeed in mouse section.
I followed it because scrollFactor is a little bit unintuitive and would make it difficult to search and autocomplete